### ISSUE TITLE
Create a configuration management service

### ISSUE THREAD

**Issue by p0000001** [gchq/Palisade#53]

**Comment by ctas582** [comment]
etcd seems to be the new ZooKeeper. You might want to look into using it instead.
[etcd vs. ZooKeeper](https://coreos.com/etcd/docs/latest/learning/why.html#zookeeper)

**Comment by p0000001** [comment]
Thanks, I'll take a look at etcd

**Comment by ac74475** [comment]
Could this be tied in to gh-28 as one of the possible cache service implementations?

**Comment by ac74475** [comment]
Is it worth creating a ConfigManagementService that would be backed by a BackingStore implementation?

**Comment by ac74475** [comment]
Blocked by gh-28, once that issue is done we can create a configManagementService backed by etcd BackedStore.

**Comment by p013570** [comment]
No longer blocked.

**Comment by ac74475** [comment]
merged into develop
